packages/framework/client-logger/app-insights-logger/package.json

```diff
@@ -91,7 +91,8 @@
 		"test": "npm run test:mocha",
 		"test:mocha": "mocha --recursive \"dist/test/**/*.spec.*js\" -r node_modules/@fluidframework/mocha-test-setup",
 		"test:mocha:verbose": "cross-env FLUID_TEST_VERBOSE=1 npm run test:mocha",
-		"tsc": "tsc"
+		"typetests:gen": "fluid-type-test-generator",
+		"typetests:prepare": "flub typetests --dir . --reset --previous --normalize"
```

Comment:
I don't think the removal of the `tsc` script was intentional, right?

Reply:
It's being complained by `flub policy check` that `tsc` has no dependency on, so I removed it.

Reply:
Hmm let's look at the error, removing this script means we would lose one of the two builds (CJS, ESM) that we currently do for this package, so it definitely doesn't seem like the way to go 😄 .

Reply:
You are right. It does result the index.d.ts output file missing. Adding it back but it fails on `flub policy check` error. Looking into it...

Reply:
I think we do need to have the `build:test` script here, since our main entry is JS file. Referring to PR: https://github.com/microsoft/FluidFramework/pull/18939